src/config.ts

```diff
@@ -0,0 +1,31 @@
+import SdkConfig from "./SdkConfig";
+
+const config = SdkConfig.get();
+
+const hiddenSplashScreenMenuItem = config['hiddenSplashScreenMenuItem'];
```

Comment:
hiddenSplashScreenMenuItem is not mentioned in feature-communicate-list.md.  I'd prefer this to be positively named and as these items aren't in a menu, perhaps call it showLoginScreenItems

Reply:
No worries, I had changed this variable too many times but I will make that change one more time according to your suggestion.

Reply:
As discussed on teams chat, loginScreen has been used as updated config.